### ISSUE TITLE
mac os  12.5 download app error

### ISSUE THREAD

**Issue by raystart** [danalite/autool#5]
![CleanShot 2023-04-21 at 08 36 36@2x](https://user-images.githubusercontent.com/25699414/233514176-7c48f46d-130b-4927-923e-993268d6f7bf.png)

**Comment by vrteee** [comment]
which version are you using? 
I guess this is most likely a network issue when downloading files from github.com

**Comment by vrteee** [comment]
```bash
✘ Failure <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)>
```

**Comment by vrteee** [comment]
Fixed in 0.1.5
